### PR TITLE
Better gravestone message, only import randint from random

### PR DESCRIPTION
**Pull request by NetSysFire**
The scripts in question only use randint from random. Importing everything is not needed. The skeleton script did not even use anything from random at all.

### CODE REVIEW

File: data/rulesets/deeds/scripts/world/objects/Weather.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # This file is distributed under the terms of the GNU General Public license.
 # Copyright (C) 1999 Al Riddoch (See the file COPYING for details).
 
-from random import *
+from random import randint
```
Comment (erikogenvik) [inline data/rulesets/deeds/scripts/world/objects/Weather.py:4]: This is unfortunately incorrect as "uniform" also is imported from "random".

File: data/rulesets/deeds/scripts/world/objects/undead/Skeleton.py [hunk 2]
```diff
@@ -1,8 +1,6 @@
 # This file is distributed under the terms of the GNU General Public license.
 # Copyright (C) 1999 Al Riddoch (See the file COPYING for details).
 
-from random import *
```
Comment (erikogenvik) [inline data/rulesets/deeds/scripts/world/objects/undead/Skeleton.py:4]: This is unfortunately incorrect as "uniform" also is imported from "random".

**Review by erikogenvik — COMMENTED** [review]
The

File: data/rulesets/deeds/scripts/world/objects/Weather.py (hunk 1 above)
Comment (NetSysFire) [reply data/rulesets/deeds/scripts/world/objects/Weather.py]: My bad. Fixed.

**Review by erikogenvik — APPROVED** [review]

**Pull request merged**